src/cmd/tools/m3ctl/main/client/checker.go

```diff
@@ -0,0 +1,21 @@
+package client
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"go.uber.org/zap"
+)
+
+func checkForAndHandleError(url string, resp *http.Response, zl *zap.SugaredLogger) error {
+	zl.Infof("resp.StatusCode:%d:\n", resp.StatusCode)
+	if resp.StatusCode > 299 {
+		dat, _ := ioutil.ReadAll(resp.Body)
+		if dat != nil {
+			fmt.Println(string(dat))
+		}
```

Comment:
you're logging the error downstream anyway, mixing prints with logging is going to result in weird output.

Reply:
moved it into the logger